src/main/java/io/gravitee/policy/rest2soap/RestToSoapTransformerPolicy.java

```diff
@@ -72,7 +74,6 @@ public void onRequest(Request request, Response response, ExecutionContext execu
     public ReadWriteStream onRequestContent(Request request, ExecutionContext executionContext) {
         return TransformableRequestStreamBuilder
                 .on(request)
-                .contentType(MediaType.TEXT_XML)
```

Comment:
Why not continuing to manage the content-type header at this level instead of putting it few lines above?